nimble/host/include/host/ble_gap.h

```diff
@@ -973,6 +975,42 @@ struct ble_gap_event {
             uint8_t adv_clk_accuracy;
         } periodic_transfer;
 #endif
+
+#if MYNEWT_VAL(BLE_POWER_CTRL)
```

Comment:
hmm syscfg.yml addition is missing (I suggest to use nimble/syscfg.yml instead of host one as eventually we will get support for this in controller too)

Reply:
Thanks for detailed comments. I have tried to address all.